core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -141,7 +141,9 @@ public void cancel() { }
     public void halfClose() { }
 
     @Override
-    public void sendPayload(Object payload) { }
+    public boolean sendPayload(Object payload) {
+      return true;
```

Comment:
We should do the same thing here and for closed calls. Right now though one returns false and the other returns true.